langchain/src/smith/runner_utils.ts

```diff
@@ -165,7 +165,7 @@ class CallbackManagerRunTree extends RunTree {
     this.callbackManager = callbackManager;
   }
 
-  async createChild(config: RunTreeConfig): Promise<CallbackManagerRunTree> {
+  createChild(config: RunTreeConfig): CallbackManagerRunTree {
```

Comment:
is this a breaking change?

Reply:
In theory if you were doing some really strange typing things or using `.then()`, but `await` (should by far be the most common way people use this) on a non async function still works so in practice people are going to be fine.

Going the other way (sync to async) would definitely be a breaking change though - we haven't cut a release yet so @dqbd are we sure we'd never want to do async operations here?